pkg/proxy/apis/config/types.go

```diff
@@ -58,6 +58,8 @@ type KubeProxyIPVSConfiguration struct {
 	// strict ARP configure arp_ignore and arp_announce to avoid answering ARP queries
 	// from kube-ipvs0 interface
 	StrictARP bool
+	// excludeExternalIP disable the ipvs forwarding from pod to LoadBalancer IP
+	ExcludeExternalIP bool
```

Comment:
this change needs to be reflected in the external types.
see https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kube-proxy/config/v1alpha1/types.go#L42

on the other hand i really think we should lock v1alpha1 at this point.
cc @kubernetes/sig-network-api-reviews 
cc @rosti @mtaufen

Reply:
good catch ! I'll add it

Reply:
hi, you should probably hold for now.

> on the other hand i really think we should lock v1alpha1 at this point.

what i meant is that we probably should not modify v1alpha1 any more and move to v1alpha2.

Reply:
ok, I'll wait for comments from others

Reply:
Unfortunately, the kube-proxy step is still nowhere near to be merged. 
@imroc let's have this in v1alpha1. It's an IPVS specific setting, so it doesn't change the proposed scheme for now. Ideally, we want to shift to use config files for now and this parameter is not present in the config file ATM (at least as the PR is).

Reply:
Yes, because I am waiting to confirm if this should be added to v1alpha1, now I have added it, PTAL @rosti

Reply:
one more commit for `make generated_files`